packages/calcite-components/src/components/notice/notice.scss

```diff
@@ -68,25 +77,27 @@
 :host {
   @apply mx-auto hidden max-w-full items-center;
   inline-size: var(--calcite-notice-width);
+  container: host / inline-size;
```

Comment:
I like `responsive-container / inline-size`. I think that would work well for every component.